### ISSUE TITLE
Confusion about the multiple arguements in target

### ISSUE THREAD

**Issue by 981Hao** [lastz/lastz#59]
Hello,
I have a question for you. If I want to do whole genome alignment between two genomes and apply the target[multiple]  in the command line, does it mean that I can parallel run the lastz? Also, I see most researchers split the genome by fasplit to parallel run the lastz, is this way equivalent to the target[multiple]? or whether I can use these two ways at the same time?
Thank you so much

**Comment by rsharris** [comment]
I'm not sure I completely understand the question.

Lastz has no direct support for parallelism. Typically whole-genome alignment is accomplished by splitting the genome into pieces and running a separate alignment job (or jobs) for each piece. Those jobs are usually run in parallel.

Splitting can be accomplished in many ways — fasplit would be one way.

What [multiple] does is tell lastz there's more than one target sequence in the input. Lastz then loads all targets into memory and aligns each query, sequentially, to those targets.

You certainly can use "target[multiple]" in any lastz jobs you run, regardless of whether those jobs are run in parallel or not.

Depending on what your genomes are, and whether they have been repeat masked, you might find it practical to split the longer chromosomes into sub-chromosomal pieces. For this it is useful to use a "subrange" sequence specifier, as described in the table at https://lastz.github.io/lastz/#seq_spec . The advantage of that (compared to using separate input files), is that lastz will report positions w.r.t. the original sequence.

**Comment by 981Hao** [comment]
Yes, that's what my question meant. Thank you for clearing it up for me
